readme.md

```diff
@@ -204,6 +204,52 @@ alfy.output([
 ]);
 ```
 
+##### options - (rerun)
+
+Type: `number`
+
```

Comment:
The docs here should link to the Alfred docs about the rerun feature.

Reply:
Okay, I will add link, thanks!